src/c++/perf_analyzer/request_rate_manager.h

```diff
@@ -88,8 +88,6 @@ class RequestRateManager : public LoadManager {
       const uint64_t measurement_window_ms, Distribution request_distribution,
```

Comment:
nit: update copyright to be correct here and elsewhere applicable in PR

Reply:
updated